css/styles.css

```diff
@@ -143,8 +145,102 @@ aside p {
   padding: 0 40px 0 20px;
 }
 
+.imageBox {
+  padding: 0px 0px 0px 0px;
```

Comment:
If all four values are the same, you only need to write it once. And in CSS, values of 0 are unitless. So we should write this: `padding: 0;`